### PR TITLE
Patch release to include source changes in Quickstart

### PR DESCRIPTION
**Pull request by fivetran-jamie**
## PR Overview
**This PR will address the following Issue/Feature:**
Patch release to include fix from https://github.com/fivetran/dbt_zendesk_source/pull/58 for https://getdbt.slack.com/archives/C01D1R2JLLA/p1733513338340409

**This PR will result in the following new package version:**

v0.19.1

**Please provide the finalized CHANGELOG entry which details the relevant changes included in this PR:**

## Bug Fixes
- In v0.19.0, Snowflake users may have seen `when searching for a relation, dbt found an approximate match` errors when running the `stg_zendesk__group_tmp` model. We have updated the `adapter.get_relation()` logic that was causing this error in the `zendesk_source` package.

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt run –full-refresh && dbt test
- [x] dbt run (if incremental models are present) && dbt test

Before marking this PR as "ready for review" the following have been applied:
- [ ] The appropriate issue has been linked, tagged, and properly assigned
- [x] All necessary documentation and version upgrades have been applied
- [x] docs were regenerated (unless this PR does not include any code or yml updates)
- [ ] BuildKite integration tests are passing
- [x] Detailed validation steps have been provided below

### Detailed Validation
Please share any and all of your validation steps:

See validation from https://github.com/fivetran/dbt_zendesk_source/pull/58

### If you had to summarize this PR in an emoji, which would it be?
 
🩹

### CODE REVIEW

**Review by fivetran-joemarkiewicz — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# dbt_zendesk v0.19.1
+[PR #180](https://github.com/fivetran/dbt_zendesk/pull/180) includes the following update:
+
+## Bug Fixes
+- In v0.19.0, Snowflake users may have seen `when searching for a relation, dbt found an approximate match` errors when running the `stg_zendesk__group_tmp` model. We have updated the `adapter.get_relation()` [logic](https://github.com/fivetran/dbt_zendesk_source/blob/main/macros/union/union_zendesk_connections.sql) that was causing this error in the `zendesk_source` [package](https://github.com/fivetran/dbt_zendesk_source).
```
Comment (fivetran-catfritz) [inline CHANGELOG.md:5]: Suggestion to link to the source release as well.

**Review by fivetran-catfritz — APPROVED** [review]
@fivetran-jamie Small suggestion but lgtm!

File: CHANGELOG.md (hunk 1 above)
Comment (fivetran-jamie) [reply CHANGELOG.md]: added

**Pull request merged**